From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Andrew Branson <andrew.branson@jolla.com>
Date: Thu, 6 Aug 2026 18:11:33 +0200
Subject: [sailfishos][ipc] Include the Qt message pump declaration

The Qt message pump is built, but message_loop.cc only includes the GTK
MessagePumpForUI declaration on Linux. Include the mutually exclusive Qt
header so TYPE_UI can instantiate its configured pump.
---
 ipc/chromium/src/base/message_loop.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/ipc/chromium/src/base/message_loop.cc b/ipc/chromium/src/base/message_loop.cc
index 82d340c55ae36..a5d6247e76264 100644
--- a/ipc/chromium/src/base/message_loop.cc
+++ b/ipc/chromium/src/base/message_loop.cc
@@ -29,6 +29,8 @@
     defined(XP_NETBSD) || defined(XP_OPENBSD)
 #  if defined(MOZ_WIDGET_GTK)
 #    include "base/message_pump_glib.h"
+#  elif defined(MOZ_WIDGET_QT)
+#    include "base/message_pump_qt.h"
 #  endif
 #endif
 #ifdef ANDROID
